test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateChatCompletionResponse.cs

```diff
@@ -59,7 +59,7 @@ internal CreateChatCompletionResponse(string id, string @object, DateTimeOffset
         /// <summary> The model used for the chat completion. </summary>
         public string Model { get; }
         /// <summary> A list of chat completion choices. Can be more than one if `n` is greater than 1. </summary>
-        public IReadOnlyList<CreateChoice> Choices { get; }
+        public IReadOnlyList<CreateChatCompletionResponseChoice> Choices { get; }
```

Comment:
This is an anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L263